_profiles/Phenotype/0.2-DRAFT.html

```diff
@@ -0,0 +1,343 @@
+      "anatomicalLocation": {
+                    "@type": "DefinedTerm",
+                    "name": "limb digit",
+                    "inDefinedTermSet": "http://purl.obolibrary.org/obo/emapa.owl",
+                    "termCode": "EMAPA_32725"
+            }
+    }
+- property: associatedDisease
+  expected_types:
+  - MedicalCondition
+  - PropertyValue
+  - URL
+  description: Disease associated to this BioChemEntity. Such a disease can be a MedicalCondition
+    or a URL. If you want to add an evidence supporting the association, please use
+    PropertyValue.
+  type: bioschemas
+  type_url: associatedDisease
+  bsc_description: ""
+  marginality: Optional
+  cardinality: MANY
+  controlled_vocab: 'Any term from a suitable ontology such as [OMIM](https://omim.org/),
+    [DO](https://disease-ontology.org/), '
+  example: |-
+    {
+      "@type": "Phenotype",
+            "associatedDisease": "https://omim.org/entry/605909?search=parkinson&highlight=parkinson"
+    }
+- property: description
+  expected_types:
+  - Text
+  description: A description of the item.
+  type: ""
+  type_url: ""
+  bsc_description: ""
+  marginality: Recommended
+  cardinality: ONE
+  controlled_vocab: ""
+  example: |-
+    {
+        "@type": "Phenotype",
+        "description": "any structural anomaly of the fingers or toes"
+    }
+- property: identifier
+  expected_types:
+  - PropertyValue
+  - Text
+  - URL
+  description: The identifier property represents any kind of identifier for any kind
+    of Thing, such as ISBNs, GTIN codes, UUIDs etc. Schema.org provides dedicated
+    properties for representing many of these, either as textual strings or as URL
+    (URI) links. See [background notes](http://schema.org/docs/datamodel.html#identifierBg)
+    for more details.
+  type: ""
+  type_url: ""
+  bsc_description: ""
+  marginality: Minimum
+  cardinality: ONE
+  controlled_vocab: ""
+  example: |-
+    //Using a text
+    {
+        "@type": "Phenotype",
+        "identifier": "MP:0002110"
+    }
+    //Using a URL directly (looks like a text)
+    {
+        "@type": "Phenotype",
+        "identifier": "http://purl.obolibrary.org/obo/MP_0002110"
+    }
+    //Using a URL modeled as schema:URL (equivalent to the previous one but more verbose)
+    {
+        "@type": "Phenotype",
+        "identifier": {
+          "@type": "URL",
+          "url": "http://purl.obolibrary.org/obo/MP_0002110"
+        }
+    }
+- property: image
+  expected_types:
+  - ImageObject
+  - URL
+  description: An image of the item. This can be a URL or a fully described ImageObject.
+  type: ""
+  type_url: ""
+  bsc_description: ""
+  marginality: Recommended
+  cardinality: ONE
+  controlled_vocab: ""
+  example: |
+    //Using a URL directly (looks like a text)
+    {
+      "@type": "Phenotype",
+      "image": "http://www.ebi.ac.uk/mi/impc/phenotype-archive/media/images/900/M00571072_00021593_download_tn_large.jpg"
+    }
+- property: isAssociatedWith
```

Comment:
This property does not exist on the corresponding type, see https://bioschemas.org/types/Phenotype/

Reply:
Is this property too generic? It is not a property that we already have defined. What does an association mean? Is this equivalent to a `rdfs:seeAlso`?

Reply:
@GiuliaBabbi what were your thoughts with the `isAssociatedWith` property?

Reply:
I think bioChemInteraction in BioChemEntity follows the same idea that isAssociatedWith in Phenotype. Both of them are broad relations that should be used if nothing more appropriate is provided.

Reply:
Actually isAssociatedWith was already a property in google sheet draft 1.0 on which I worked during the F2F, I just switched the marginality to recommended. In my mind, it was the preferential way to associate a BioChemEntity (genes, proteins..)

Reply:
The previous version of Phenotype was done during the BioHackathon Europe 2018, maybe @ficolo remembers something about isAssociatedWith?

Reply:
gene-phenotype association is the way it will be described by people working with phenotypes. We are changing it to bioChemAssociation in case it helps, similar to the idea behind bioChemInteraction for the BioChemEntity type. And we improved the description